lte/gateway/deploy/roles/ovs_prepare/tasks/main.yml

```diff
@@ -11,124 +11,105 @@
 # limitations under the License.
 #
 
-- fail: msg="MAGMA_ROOT and OUTPUT_DIR are required"
-  when:
-    - MAGMA_ROOT is not defined or OUTPUT_DIR is not defined
+- name: validate MAGMA_ROOT env variable.
+  fail:
+    msg: MAGMA_ROOT is required
+  when: MAGMA_ROOT is not defined
 
-- name: Get Kernel version
-  shell: uname -r
-  register: KVERS
+- name: validate kernel version supported.
+  fail:
+    msg: Kernel {{ ansible_kernel  }} not supported please use 4.9.0-9-amd64
+  when: ansible_kernel != '4.9.0-9-amd64'
 
-- fail: msg="Kernel Version {{ KVERS.stdout  }} not supported please use 4.9.0-9-amd64"
```

Comment:
I am not sure why do we need changes to the ansible role tasks. Can we separate out changed to task and unit tests?

Reply:
Those changes are needed to pass the Molecule idempotence and lint checks. maybe I can disable them to reduce the number of changes.

Reply:
Can we disable them for now? I think our whole ansibles roles need a complete restructure. 
While we restructure them we'll be able to tackle the indempotence and lint changes.

Reply:
I'm not sure if it's possible to disable them. I can investigate more about it.